tests/unit/model/rollback-attributes-test.js

```diff
@@ -188,12 +202,17 @@ test("a deleted record's attributes can be rollbacked if it fails to save, recor
     person.save().then(null, function() {
       assert.equal(person.get('isError'), true);
       assert.equal(person.get('isDeleted'), true);
+      assert.equal(person.get('rolledBackCount'), 0);
+
       run(function() {
         person.rollbackAttributes();
       });
+
       assert.equal(person.get('isDeleted'), false);
       assert.equal(person.get('isError'), false);
       assert.equal(person.get('hasDirtyAttributes'), false, "must be not dirty");
+      // Should `rolledBack` get triggered when a record is deleted and `rollbackAttributes` is called?
+      assert.equal(person.get('rolledBackCount'), 0);
```

Comment:
Should the `rolledBack` event get triggered when a deleted record calls `rollbackAttributes`?